### DISCUSSION TITLE
Understanding gene usage

### DISCUSSION THREAD

**Discussion by benjytan88** [milaboratory/mixcr#1156] [Q&A] [answered]
Hi MiXCR developers,
I wanted to ask what is the meaning of multiple TRBV genes detected in a sample? For example, what is the meaning of TRBV3-1, TRBV14, TRBV3-2, TRBV4-2 in sample K169_N1t1 in the table below? 

![Screenshot 2023-06-15 at 15 22 45](https://github.com/milaboratory/mixcr/assets/24598265/04887a1b-789b-4b12-839f-36e9a062c6d9)

**Comment by PoslavskySV** [comment]
Hi,  could you please elaborate how did you produce this table? It does not look like output from MiXCR.

**Comment by benjytan88** [comment]
Hi @PoslavskySV, that was a table which summarize gene usage in all my samples. Here's the output from MiXCR. 

![Screenshot 2023-06-15 at 16 13 00](https://github.com/milaboratory/mixcr/assets/24598265/49c1be0e-6642-47a8-ab6a-6de94bc09c53)

As you can see, there are several rows which have more than 1 V gene called. I would like to understand what are these and how to interpret these.

**Comment by PoslavskySV** [comment]
Thanks! Could you please paste exact commands you used with MiXCR?  The names of the columns look quite different from the standard MiXCR output.

**Comment by benjytan88** [comment]
Command as below. This data is from bulk RNA-seq and MiXCR v4.3.2 was used. 

`mixcr analyze rnaseq-full-length --species hsa ./K548-M20_R1_001.fastq.gz ./K548-M20_R2_001.fastq.gz ./K548-M20`

**Comment by PoslavskySV** [comment, marked as answer]
Did you rename columns manually? The column "V.name" is probably "AllVHitsWithScore" from MiXCR output. This column contain all V hits with corresponding alignment scores. E.g. the value in cell:

```
TRBV12-3*00(240), TRBV12-4*00(240) 
```

means that the read was aligned to two V genes: TRBV12-3  and TRBV12-4 with equal alignment score  (240). You can use "bestVHit" or "bestVGene" or "bestVFamily" to have only one (best) hit in the output (see [export docs](https://mixcr.com/mixcr/reference/mixcr-export)).

**Comment by benjytan88** [comment]
I see. OK, I will try and see. 
I didn't rename them manually, but the table was obtained by loading the MiXCR results into R using the `immunarch` package.
Maybe there's some renaming done from there...?

**Reply by PoslavskySV** [reply]
Yes, this is 100% due to immunarch. You need to check how it converts columns.
